src/components/backgrounds/BackgroundPreviewSettings.tsx

```diff
@@ -190,7 +201,8 @@ const BackgroundPreviewSettings = ({
               ? l10n("FIELD_PREVIEW_AS_SCENE", {
                   sceneName: sceneName(scene, sceneIndex),
                 })
-              : l10n("FIELD_PREVIEW_AS_DEFAULT")}
+              : ([l10n("FIELD_PREVIEW_AS_MONO"), l10n("FIELD_PREVIEW_AS_DEFAULT")][+value])
```

Comment:
As someone who has never used TypeScript before, I found the `[+value]` syntax a little strange, but it's what StackOverflow told me to do. I'm considering revisiting the weird stringified index hack when I add SGB preview support anyway, but this is a little too hacky for your tastes now, let me know.